### PR TITLE
Crypto changes

### PR DESCRIPTION
**Pull request by Michael-Stanford**
Changed the UUID generation to use the Node native crypto module. This is in response to Alex Allain's concern that the uuid module is not cryptographically strong.

### CODE REVIEW

**Comment by Michael-Stanford** [comment]
I have also made this change in cares-reporter but not pushed it to GitHub.

File: packages/client/src/store/modules/grants.js [hunk 1]
```diff
@@ -24,10 +24,10 @@ export default {
   },
   actions: {
     fetchGrants({ commit }, {
-      currentPage, perPage, orderBy, searchTerm, interestedByMe, assignedToMe,
+      currentPage, perPage, orderBy, searchTerm, interestedByMe, assignedToMe, aging,
     }) {
       const query = Object.entries({
-        currentPage, perPage, orderBy, searchTerm, interestedByMe, assignedToMe,
+        currentPage, perPage, orderBy, searchTerm, interestedByMe, assignedToMe, aging,
```
Comment (Rapol) [inline packages/client/src/store/modules/grants.js:30]: unfortunately still some left over code from steve branch

File: packages/client/src/views/MyGrants.vue [hunk 2]
```diff
@@ -6,6 +6,9 @@
     <b-tab title="Assigned">
       <GrantsTable :showAssigned="true"/>
     </b-tab>
+    <b-tab title="Aging">
+      <GrantsTable :showAging="true"/>
+    </b-tab>
```
Comment (Rapol) [inline packages/client/src/views/MyGrants.vue:11]: here

File: packages/server/src/db/index.js [hunk 3]
```diff
@@ -222,6 +232,12 @@ async function getGrants({
                         if (filters.assignedToUser) {
                             qb.where(`${TABLES.assigned_grants_user}.user_id`, '=', filters.assignedToUser);
                         }
+                        if (filters.aging) {
```
Comment (Rapol) [inline packages/server/src/db/index.js:235]: and here

**Review by Rapol — CHANGES_REQUESTED** [review]

**Pull request closed without merging**